src/components/base-label/style.scss

```diff
@@ -1,9 +1,3 @@
-.label {
-  color: #42b983;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-}
 .logo {
```

Comment:
Can we remove this? Does UI toolkit gives a way to display logo? If not, we should consider removing logo all together.

Reply:
Alternative for this is to all height and width inline styles. styles-"width: 10px; height:10px". https://quasar.dev/vue-components/img#qimg-api. @jjalan

Reply:
@jjalan Updated!